app/src/main/res/layout/fragment_pledge.xml

```diff
@@ -256,6 +256,13 @@
             android:layout_height="wrap_content"
             android:layout_weight=".1">
 
+            <LinearLayout
```

Comment:
I'm confused. Why do we need this container? Are we not hiding the whole section?

Reply:
The `+` icon is not being hidden while the shipping rules are loading, as a result, we can see icon close to the shipping_amount_loading_view`. So to solve this, I put the ImageView and TextView  in a `LinerLayout` to hide/show the icon and amount all at once.

Reply:
ok i think i get it!